examples/01-blog/Blog/Type/Field/HtmlField.php

```diff
@@ -24,8 +28,8 @@ public static function build(string $objectKey): array
                 ],
                 'maxLength' => Types::int(),
             ],
-            'resolve' => static function ($object, $args) use ($objectKey) {
-                $html = $object->{$objectKey};
+            'resolve' => static function ($value, $args) use ($config) {
+                $html = $config['resolve']($value, $args);
```

Comment:
I would extract `$config['resolve']` outside of this callback - slight performance boost to not perform the array lookup multiple times

Reply:
Is this what you mean? 942370f7